### ISSUE TITLE
Payee account parameter cannot be empty. Check perfectmoney config file for details.

### ISSUE THREAD

**Issue by msurubel** [mohsennrs/perfectmoney-laravel#1]
**Payee account parameter cannot be empty. Check perfectmoney config file for details.**'
Config file is ok, but showing this text. How to solved it

**Comment by mohsennrs** [comment]
You need to publish the config file and then put the value "PM_PAYEE_ACCOUNT" in the config file or in the .env file. If you've done this step and still getting the error please let me know.

**Comment by msurubel** [comment]
Yes I am already set my payee Account "UA64456" in "PM_PAYEE_ACCOUNT" but still showing this error.

**Comment by mohsennrs** [comment]
I can't replicate the error with the 'pm_payee_account' field being set. Can you please copy the content of 'perfectmoney.php' file and '.env' file here? You can leave out sensitive information.
